README.md

```diff
@@ -6,4 +6,33 @@ Welcome to the GitHub Workflows repository of AI-CFIA! This repository contains
 
 This repository is dedicated to providing automated solutions and workflows for various operations within our GitHub projects. It contains a collection of reusable workflows for the CI/CD pipelines, testing, and other automated processes within GitHub Actions.
 
-Find out how to use these workflows [here](./github-workflows-documentation.md)
+Each workflow has an associated documentation file that explains its purpose, usage, and any required secrets or inputs. You can find these files in the same directory as their corresponding workflow file.
+
+## Workflow Documentation
+
+Here are the available workflows and their associated documentation:
+
+- [Reusable Docker Container Build](./workflow-build-container.md)
+- [Reusable Docker Container Deployment to Google Cloud Run](./workflow-deployment-gcr.md)
+- [Reusable Lint and Test Workflow for Node (React) Projects](./workflow-lint-test-node.md)
+- [Reusable Lint and Test Workflow for Python Projects](./workflow-lint-test-python.md)
+- [Reusable Workflow for Standardization Validation](./workflow-standardization-validation.md)
+- [Reusable Build and Deploy to GitHub Pages](./workflow-build-deploy-ghpages.md)
+
+## Usage
+
+To use these workflows in your project, you can call them using `workflow_dispatch`. Here’s a basic example of how to call a workflow:
+
+```yaml
+name: My Workflow
```

Comment:
even in examples, take the habit of having good names so that we don't end up with developers copy/pasting "My Workflow" in their new repos.

Reply:
done

Reply:
provide more guidance on the naming fo the workflow.

Reply:
like a whole section just for the title?

Reply:
I changed the example's title if that's what you meant but not sure if that's good for you